pkg/probe/http/http.go

```diff
@@ -129,6 +129,10 @@ func DoHTTPProbe(url *url.URL, headers http.Header, client GetHTTPInterface) (pr
 	if res.StatusCode >= http.StatusOK && res.StatusCode < http.StatusBadRequest {
 		if res.StatusCode >= http.StatusMultipleChoices { // Redirect
 			klog.V(4).Infof("Probe terminated redirects for %s, Response: %v", url.String(), *res)
+			// add some message in case body is empty when redirect is terminated
+			if len(body) == 0 {
+				return probe.Warning, "Probe terminated redirects", nil
+			}
 			return probe.Warning, body, nil
```

Comment:
actually thinking about it more I think this can be improved by adding "Probe terminated redirects, Response body: %v". wdyt?

Reply:
I will lgtm as-is though, but improving the error message may be helpful. Nobody should depend on exact message. But keeping the response body is important for troubleshooting.

Reply:
It sounds better.